Week1/Section1/Exercise10/Source10.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+Exercise 10 - Swapping
+Write a program that will swap the value of two variables
+
+Example output
+Please enter two numbers: 6 5
+
+Your first number: 6
+Your second number: 5
+
+Your first number swapped: 5
+Your second number swapped: 6
+*/
+
+#include <iostream>
+
+using namespace std;
+
+int main() {
+
+	int num1, num2, aux;
```

Comment:
Always try to declare your variables as close to their definition as possible. num1 and num2 are fine up here since they're assigned in the next lines (though they could still be closer to where they are assigned). But there is no reason to have aux all the way up here if it is used later. Your line 30 could instead be written as:
```
int aux = num1;
```